django_object_actions/templates/django_object_actions/action_trigger.html

```diff
@@ -0,0 +1,22 @@
+{% load add_preserved_filters from admin_urls %}
+
+{% if tool.button_type == 'a' %}
+  <a href="{% add_preserved_filters action_url %}"
+    title="{{ tool.standard_attrs.title }}"
+    {% for k, v in tool.custom_attrs.items %}
+      {{ k }}="{{ v }}"
```

Comment:
need to look into if we need to make sure the html is safe (will be one of the things I experiment with)
```suggestion
      {{ k }}="{{ v|escape }}"
```

https://docs.djangoproject.com/el/4.2/ref/templates/builtins/#escape

Reply:
version before had this line without escape, here: https://github.com/crccheck/django-object-actions/pull/168/files#diff-7b01c579f97b840216ab2dc0fb68a59712317af56efac2504ed9edcf021480baL10